draft-wang-ppm-differential-privacy.md

```diff
@@ -193,13 +205,18 @@ values they produce for the Collector).
 In this document, we shall refer to the composition of DP mechanisms into a
 scheme that provides (some notion of) DP as a "DP policy". For some policies,
 noise is added only by the Clients or only by the Aggregators, but for others,
-noise might be added by both Clients and Aggregators.
+noise might be added by both Clients and Aggregators. The best choice of policy
+depends one's "trust model": if most Clients can be trusted to noise their
+measurements correctly, then a high degree of privacy can be achieved with
+minimal loss in utility; but if the trust model includes the possibility that
+many Clients are corrupted by the attacker, then the same degree of privacy can
+only be achieved at the cost of decreased utility.
```

Comment:
To me this section indicates one must define the trust model of their deployment then choose DP policy. But in reality the trust model may not hold true all the time. It may not be possible for user to choose a trust model and that model's best policy. For e.g. one may not know how many clients are honest. In reality, user probably will choose a DP policy then analyse/explain their DP guarantee under different trust model. Maybe we can word this to something like the choice of DP policy will be affected by the trust model, etc. @cjpatton

Reply:
Great idea! I pruned some of the text here and implemented your suggestion in the itemized list below, PTAL.